### ISSUE TITLE
Fine-tuned visual and audio accessibility options

### ISSUE THREAD

**Issue by jae1911** [Yellow-Dog-Man/Resonite-Issues#1213]
### Is your feature request related to a problem? Please describe.

While in game, there are some systems (E.G audio links, meta-memes) producing strobing light effects, "flashbangs" or even loud sounds.
Those can be annoying, cause discomfort or be dangerous (in the case of epilepsy).

### Describe the solution you'd like

It would be nice to have something in the same vein as #554, a settings system to fine tune more what you would see and hear, all the users needing to do is placing a component on their systems.

For instance, there could be components and settings for (but not limited to):
- Strobing lights
- Bright lights
- Loud noises
- (maybe someone will have more ideas in the comments of this issue)

It would still rely on users being good players and adding the components to their creations, but would make hiding those kinds of content trivial.

### Describe alternatives you've considered

Get everybody to agree on standards that could get disabled locally by anybody (has the caveat of using a cloud variable or other systems which could expose such user has such setting on or off, which in term could enable targeted harassment as underlined in #554).

### Additional Context

This issue is closely related to #554.
Would also need to wait for #584.

I opened this one with the same thought as @kulzae's comment on #554 given that issue seems to be more related to animations/motion than strobing lights or other sources of discomfort (also sorry for the ping; you might find this interesting).

**Comment by JackTheFoxOtter** [comment]
There is an issue with this being a preference you can determine in-world. I understand the intention, but this could be used by malicious actors to specifically target people that are sensitive to those kinds of effects. Of course that would be a moderation situation, but the damage might already be done. It would also potentially be a privacy concern depending on the circumstances.

I feel like any system like that would better be fully transparent to the world so other users can't identify your preferences. That would also mean no extra work has to be put in on the creator's side of things to respect those preferences.

**Comment by jae1911** [comment]
I agree that an agnostic implementation would be better, but then, would require more complex systems behind to detect those situations reliably.

Providing at least a basic filtering system would already be better than now (given literally nothing exists), even though it requires creators to be good players (and I trust most of them are right now given the current size and friendliness of the community).
Also as mentioned in #554, all of this would be completely opaque to the other players, though I do not know any more specific details on how that would work.
